tests/test_demo_script.py

```
@@ -0,0 +1,75 @@
+    "routine_l1_continues": True,
+    "dangerous_downgrade_blocked": True,
+    "text_redaction_ok": True,
+    "binary_evidence_fail_closed": False,
+    "agent_install_ok": True,
+    "strict_dep_ok": True,
+    "verdict": "FAIL",
+}) + "\\n", encoding="utf-8")
+raise SystemExit(1)
+""",
+            )
+            completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)